spec/bookingsync/api/client/rentals_spec.rb

```diff
@@ -65,6 +65,16 @@
     it "returns a single rental" do
       rental = client.rental(prefetched_rental[:id])
       expect(rental.name).to eql(prefetched_rental[:name])
+      expect(rental).to_not have_key(:availability)
+    end
+
+    context "with additional query params" do
+      it "returns a single rental with defined params" do
+        rental = client.rental(prefetched_rental[:id], include: [:availability, :change_over])
+        expect(rental.name).to eql(prefetched_rental[:name])
```

Comment:
Actually it's mixed, even in this particular file both forms are used. Ok, creating separate issue then.